components/app-proxy/get-access-token.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Change account location to keep roles for later steps
+export AZURE_CONFIG_DIR=~/.azure-app-proxy-user
+
+# Fetch params from terraform data source
+eval "$(jq -r '@sh "username=\(.username) password=\(.password)"')"
```

Comment:
eval shouldn't be necessary, it's very dangerous and not something i would expect to see in a script

Reply:
Hm, I just took this from the example at https://registry.terraform.io/providers/hashicorp/external/latest/docs/data-sources/external#processing-json-in-shell-scripts -- I can revisit this though if it's still unexpected here @timja ?

Reply:
hmm i guess =/, fine for now